src/rpc/net.cpp

```diff
@@ -671,7 +672,6 @@ static UniValue listbanned(const JSONRPCRequest& request)
         rec.pushKV("address", entry.first.ToString());
         rec.pushKV("banned_until", banEntry.nBanUntil);
         rec.pushKV("ban_created", banEntry.nCreateTime);
-        rec.pushKV("ban_reason", banEntry.banReasonToString());
```

Comment:
b6a34484 Should also remove `ban_reason` from the help at line 650, I believe.

Reply:
Done.

Reply:
Have the release notes been updated with this breaking RPC change?

Reply:
Fixed.